src/index.html

```diff
@@ -1,13 +1,375 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBike</title>
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/logo.svg">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo logo--white">
+              <img 
+                src="images/logo.svg"
+                alt="MyBIKE white logo"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="header__links">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+                <div class="icon__tooltip">+1 234 555-55-55</div>
+              </a>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="logo">
+              <img 
+                src="images/logo-black.svg"
+                alt="MyBIKE black logo"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="menu__links">
+              <a href="#" class="icon icon--close"></a>
+            </div>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55" class="menu__call menu__call--phone"
+          >
```

Comment:
You need to fix format phone only in href everywhere
```suggestion
          <a href="tel:+12345555555" class="menu__call menu__call--phone">
```

Reply:
Not sure if it's the mistake but seems like it
phone number in header and in footer are different

Reply:
Not sure if it's the mistake but seems like it phone number in header and in footer are different
Should i change it?

Reply:
The same as in figma design